features/cellular/framework/AT/AT_CellularNetwork.cpp

```diff
@@ -485,6 +496,12 @@ nsapi_error_t AT_CellularNetwork::get_signal_quality(int &rssi, int &ber)
         return NSAPI_ERROR_DEVICE_ERROR;
     }
 
+    if (rssi == 99) {
```

Comment:
Although this was already integrated, during the rollup review, these numbers were questions - I assume rssi is a value 0-99, but then what below means, on line 502 might help to add a comment

Reply:
@0xc0170 RSSI value is always in dBm, and actual range is -51 ... -113 dBm Previously returned values 0..99 were raw values from AT protocol spec due to conversion implementation was missing.

Reply:
That explains it, however we have in the code magic values that do not , either create defines or add a comment describing this calculation. You can send a new PR

Reply:
@0xc0170 thx, I will fix that within another PR.